pkg/kafka/ingester/ingester.go

```diff
@@ -51,8 +51,8 @@ type Config struct {
 
 // RegisterFlags registers the flags.
 func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
-	cfg.LifecyclerConfig.RegisterFlagsWithPrefix("kafka-ingester", f, util_log.Logger)
-	cfg.PartitionRingConfig.RegisterFlags(f)
+	cfg.LifecyclerConfig.RegisterFlagsWithPrefix("kafka-ingester.", f, util_log.Logger)
+	cfg.PartitionRingConfig.RegisterFlagsWithPrefix("kafka-ingester.", f)
```

Comment:
We should settle on either `kafka-ingester` or `kafka-ingestion` everywhere. `ingester.goL135` in this PR uses `ingestion`

Reply:
This is actually the new storage ingester. I think we'll find better name once we re-active that back and those are not used anymore.